step.yml

```diff
@@ -122,9 +123,19 @@ inputs:
         - if `fastlane_version` input not specified, latest version will be used
   - fastlane_version: "latest"
     opts:
+      category: Debug
       title: "Fastlane version"
       summary: "Install a specific version of the `fastlane` gem."
       description: |-
         This option lets you specify a specific version of the `fastlane` gem to be installed.
-
+  - options:
+    opts:
+      category: Debug
+      title: "Additional options for `match` call"
+      description: |-
+        Options added to the end of the `match` call.
+        
+        If you want to add more options, list them separated by a space character.
+        
+        Example: `--team_name`
```

Comment:
how do you add an arg that takes a value??

Reply:
I'm honestly not sure … but from looking at other implementations, it could work something like "-- team ABC123DEF --readonly". I don't now how to test this, though.

Reply:
it uses `shellquote.split`, I think it should work :)